From ab43f0b744ef2b9c1ba1778844f24d78e33575ea Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Wed, 23 Feb 2022 11:00:12 -0800
Subject: tf: log CPU info on test run (#37493)

The e2 nodes we use get a ~random CPU type. Add logging so we can see if
this correlates to flakes
---
 prow/integ-suite-kind.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/prow/integ-suite-kind.sh b/prow/integ-suite-kind.sh
index 468c8648ea..63b3e6f10b 100755
--- a/prow/integ-suite-kind.sh
+++ b/prow/integ-suite-kind.sh
@@ -101,6 +101,9 @@ while (( "$#" )); do
   esac
 done
 
+echo "Checking CPU..."
+grep 'model' /proc/cpuinfo
+
 # Default IP family of the cluster is IPv4
 export IP_FAMILY="${IP_FAMILY:-ipv4}"
 
-- 
2.35.3

